operations/mimir-tests/test-autoscaling-generated.yaml

```diff
@@ -928,8 +928,8 @@ spec:
           limits:
             memory: 24Gi
           requests:
-            cpu: "1"
```

Comment:
With all the type troubles we've had, we should consider having a test that makes sure we get the right result when cpu requests are a string (like this).